auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div id="entity-management">
```

Comment:
Used in AddBusinessDialog to attach to. This is not necessary but is a good practice IMHO.